src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer it is work");
+    }
+
+    public void stopWork() {
+

```

Comment:
needless empty line. method body should not begin with empty line. fix in all places